build.gradle

```diff
@@ -4,11 +4,15 @@ subprojects {
     buildscript {
         repositories {
             jcenter()
+            maven {
```

Comment:
nit: these 3 lines can be replaced by `gradlePluginPortal()`
If it is not available, we can update gradle version to `4.4.1` to replace it. 

Can be done later too.

Reply:
done 👍